pkg/controllers/provisioning/provisioner.go

```diff
@@ -115,6 +112,9 @@ func (p *Provisioner) Reconcile(ctx context.Context, _ reconcile.Request) (resul
 	if triggered := p.batcher.Wait(ctx); !triggered {
 		return reconcile.Result{}, nil
 	}
+	if !p.cluster.Synced(ctx) {
+		return reconcile.Result{RequeueAfter: time.Second}, nil
```

Comment:
Same here on the reconciling. Can we just return `Requeue: true` and rely on the in-memory backoff?

Reply:
Makes sense 👍